### ISSUE TITLE
Unable to import WithLockInterface interface

### ISSUE THREAD

**Issue by mayorovad** [Kikobeats/superlock#11]
Hello! Trying to use withLock function in some class:

```typescript
import { withLock, WithLockInterface } from "superlock";


class SomeClass {
    lock: WithLockInterface;

    constructor() {
        this.lock = withLock(1);
    }
}
```
Got error `Module '"superlock"' declares 'WithLockInterface' locally, but it is not exported.`. 

I think `WithLockInterface` should be exported as it's part of library API.

**Comment by Kikobeats** [comment]
Fixed at `1.2.2`, can you try it? 🙂

**Comment by mayorovad** [comment]
Yes, all good now! Thank you 😸
